pkg/api/store/workload/aggregate_store.go

```diff
@@ -213,17 +213,15 @@ func capabilitiesToUpperCase(data map[string]interface{}) map[string]interface{}
 	containers := convert.ToMapSlice(data["containers"])
 	elements := []string{"capDrop", "capAdd"}
 
-	if containers != nil {
-		for _, c := range containers {
-			for _, element := range elements {
-				caps := convert.ToStringSlice(c[element])
-				newCaps := []string{}
-				if caps != nil {
-					for _, cap := range caps {
-						newCaps = append(newCaps, strings.ToUpper(cap))
-					}
-					c[element] = newCaps
+	for _, c := range containers {
+		for _, element := range elements {
+			caps := convert.ToStringSlice(c[element])
+			newCaps := []string{}
+			if caps != nil {
```

Comment:
@teresy should check on line 220 be removed too?

Reply:
I thought about it, but was unsure. The reason is, the check on line 220 wraps not only the `for` loop starting on line 221, but also the line `c[element] = newCaps`. So to safely remove the check on line 220, we should think about the effect of `caps` being `nil` on `c[element] = newCaps`. It *looks* like this is OK, but maybe a bit overzealous. The outer if-statement can be removed for sure though, since it wraps the entire for loop.

Reply:
ok, good point. lets leave it the way it is for now